esp/src/src/react/components/Frame2.tsx

```diff
@@ -98,32 +98,38 @@ const useStyles = makeStyles((theme) => ({
 }));
 
 interface Frame2 {
-    widgetClassID: string,
-    widgetParams?: RouteParams
 }
 
-export const Frame2: React.FunctionComponent<Frame2> = ({
-    widgetClassID = "ActivityWidget",
-    widgetParams
-}) => {
+export const Frame2: React.FunctionComponent<Frame2> = () => {
     const classes = useStyles();
+
     const mainRef = React.useRef<HTMLDivElement>();
 
+    const [body, setBody] = React.useState(<h1>...loading...</h1>);
```

Comment:
I added the circle progress previously but it was removed? https://github.com/GordonSmith/HPCC-Platform/blob/98347ee87a3109654ad824eb6fc52bff9ed8a1fd/esp/src/src/react/components/Frame.tsx#L69

Reply:
Yes - was trying to get back to basics for the sizing.  Adding it back should be easy enough.